### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by krisdo**
-Added title to Getting Around info functionality
-Added custom marker to Map Component
-Added styling to More About Location button

### CODE REVIEW

File: client/components/map.jsx [hunk 1]
```diff
@@ -33,10 +32,17 @@ class Map extends Component {
   }
 
   createMarker (lat, lng){
+    
+    const markerIcon = {
+      url: `${window.location.href}airbrb_home.png`,
+      scaledSize: new google.maps.Size(62,62),
```
Comment (cmonez) [inline client/components/map.jsx:38]: What does this mean
??

File: client/app.jsx [hunk 2]
```diff
@@ -80,10 +87,10 @@ class Location extends React.Component {
     return(
       <div>
         <h2>Location</h2>
-        {(this.state.lat !== null && this.state.lng !== null) ?  <Map id={'map'} style={{zIndex: -1, position: 'relative', width: '100%', height: 300, borderRadius: '12px'}} lat={this.state.lat} lng={this.state.lng}/> : <div>loading...</div>}
+        {(this.state.lat !== null && this.state.lng !== null) ?  <Map id={'map'} style={{zIndex: -1, position: 'relative', width: '100%', height: 300}} lat={this.state.lat} lng={this.state.lng}/> : <div>loading...</div>}
```
Comment (cmonez) [inline client/app.jsx:90]: I see the zIndexing here haha

File: client/components/map.jsx (hunk 1 above)
Comment (krisdo) [reply client/components/map.jsx:38]: it's to make the custom marker smaller to that dimension

File: client/app.jsx (hunk 2 above)
Comment (krisdo) [reply client/app.jsx:90]: yeah, if I don't do it then Andy's modal is going to be behind my map...

File: client/components/map.jsx (hunk 1 above)
Comment (cmonez) [reply client/components/map.jsx:38]: INteresting interesting.

**Pull request merged**